src/ngraph/op/fused/prelu.cpp

```diff
@@ -0,0 +1,81 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "ngraph/op/fused/prelu.hpp"
+
+#include "ngraph/op/add.hpp"
+#include "ngraph/op/broadcast.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/convert.hpp"
+#include "ngraph/op/greater.hpp"
+#include "ngraph/op/less.hpp"
+#include "ngraph/op/multiply.hpp"
+#include "ngraph/op/reshape.hpp"
+#include "ngraph/op/util/broadcasting.hpp"
+#include "ngraph/util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+op::PRelu::PRelu(const shared_ptr<Node>& data, const shared_ptr<Node>& slope)
+    : FusedOp("PRelu", {data, slope})
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::PRelu::decompose_op() const
+{
+    auto data = get_argument(0);
+    auto data_shape = data->get_shape();
+    auto slope = get_argument(1);
+    auto slope_shape = slope->get_shape();
+
+    if ((slope_shape.size() == 1) && (slope_shape.at(0) != 1))
+    {
+        auto it = std::find(std::begin(data_shape), std::end(data_shape), slope_shape.at(0));
+        auto index = std::distance(std::begin(data_shape), it);
+        slope = make_broadcast_node(slope, data->get_shape(), index);
+    }
+    else if (data_shape != slope_shape)
+    {
+        slope = numpy_style_broadcast({slope, data})[0];
+    }
+
+    // x <  0 => f(x) = x * slope
+    // x >= 0 => f(x) = x
+
+    std::shared_ptr<ngraph::Node> zero_node = std::make_shared<ngraph::op::Constant>(
+        data->get_element_type(), ngraph::Shape{}, std::vector<double>{0});
```

Comment:
Why not create const node with desired shape directly ?
```suggestion
        data->get_element_type(), data->get_shape(), std::vector<double>{0});
```